### ISSUE TITLE
2.3.8 patch release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#1785]
- #1754 (should have been included in 2.3.7)
- #1764 
- #1784 
- #1773
- #1788
- #1791

**Comment by joeparsons** [comment]
## Pick list

```
git cherry-pick 9701b9a4c7a72b81abd46552603d9fe09315eab6
git cherry-pick df84b7dfc141d46c2cda1e3251cd82934eb8d698
git cherry-pick 39e4a6304095a890336c96805bd705f089b057cd
git cherry-pick c1a57201b56588053179be1f1a5b01179406e37b
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.3.7..HEAD --pretty='format:- %s'`

```
- Closes #1790, Closes #1792: Update Pathauto patterns for Drupal 9.3+ and use drupal/ctools:4.0.1. (#1791)
- Closes #1382: Add role_delegation patch for restricting access to cancel/remove admin users (#1773)
- Closes #1782: Add mesa as background color option to cards, text on background, and split screen. (#1784)
- Update drupal/date_ap_style requirement from 1.4 to 1.5.0 (#1764)
- Closes #1743 Fix block embeds (#1754)
- Closes #1787: Restore missing Drupal core backbone-min.js.map file (2.3.x only) (#1788)
```

**Comment by joeparsons** [comment]
Release created https://github.com/az-digital/az_quickstart/releases/tag/2.3.8
